addons/pos_restaurant/static/src/overrides/components/product_screen/actionpad_widget/actionpad_widget.xml

```diff
@@ -34,7 +34,7 @@
         <xpath expr="//button[hasclass('validation')]" position="before">
             <button t-if="this.swapButton"
                 t-on-click="() => pos.pay()" 
-                class="button pay-order-button btn btn-lg rounded-0 py-3 fs-6"
+                class="button pay-order-button btn btn-lg rounded-0 py-3 fs-6 border-bottom"
```

Comment:
```suggestion
                class="button pay-order-button btn btn-lg rounded-0 py-3 fs-6 border-top border-bottom"
```
![image](https://github.com/odoo/odoo/assets/100123534/d0f75ec5-d6d0-4518-81e5-39bb5f3dd0ee)

I would like to add border-top for payment button also.